simtools/util/names.py

```diff
@@ -667,6 +667,8 @@ def corsikaConfigFileName(arrayName, site, zenith, viewCone, label=None):
         Array name.
     site: str
         Paranal or LaPalma.
+    primary: str
+        Primary particle (e.g gamma, proton etc).
```

Comment:
Is there a way in the documentation to link to the yaml file with the options to make it clear for the user? Just curious, not urgent at all.